src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3247,6 +3247,9 @@ internal bool EnableEnumArrayBlockInitialization
             }
         }
 
+        public override INamespaceDeclaration DeclarationRoot
+            => Declarations.GetDeclarationRoot(this);
```

Comment:
Possibly "GlobalDeclaration" given that we have Compilation.GlobalNamespace.  not too important, but since this is a public API we should try to land on what we think is best here.

Reply:
Hmm, I'm not sure, global namespace is a thing whereas global declaration isn't. Thoughts? @dotnet/compiler-api

Reply:
it's a good question.  it might make sense to synthesize in that case?  i'm not actually sure what would be best :)